src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -1307,9 +1314,9 @@ def call(
         )
 
     # Copied from transformers.models.bert.modeling_tf_bert.TFBertForTokenClassification.serving_output
-    def serving_output(self, output):
-        hs = tf.convert_to_tensor(output.hidden_states) if self.config.output_hidden_states else None
-        attns = tf.convert_to_tensor(output.attentions) if self.config.output_attentions else None
+    def serving_output(self, output: TFTokenClassifierOutput) -> TFTokenClassifierOutput:
+        hs = tf.convert_to_tensor(value=output.hidden_states) if self.config.output_hidden_states else None
+        attns = tf.convert_to_tensor(value=output.attentions) if self.config.output_attentions else None
```

Comment:
I don't understand why we're putting named arguments everywhere, even when they're not named arguments. For example here, I don't think it makes it any easier to read `tf.convert_to_tensor` if you specify `value`, on the contrary. 

It makes it seem like this method could (or should) take other arguments, while it is not the case: it should take a single argument, and named parameters can be used to slightly modify the behavior.

Anyway this is a nit, as it doesn't change anything and you've already switched everything, but I don't understand this choice

Reply:
What you are saying makes sense, I will rework this part of the PR 👍